### ISSUE TITLE
`_get_comp_words_by_ref` not found

### ISSUE THREAD

**Issue by srsudar** [imomaliev/tmux-bash-completion#3]
This emerged from [a discussion on a PR](https://github.com/scop/bash-completion/pull/81) at the `bash-completion` repo.

When I start up bash and source the completions file I get an error when trying to tab complete. I type `tmux at` and hit `<tab>`, expecting for it to show me the options starting with `attach`. Instead I get the error: `bash: _get_comp_words_by_ref: command not found` added to my command line.

I use `zsh` these days over `bash`, so it is possible something has gone wonky with my `bash` setup. I can tab-complete other things.

The full terminal session is below.

```
bash-4.2$ ls
README.md       completions
bash-4.2$ source completions/tmux
bash-4.2$ tmux atbash: _get_comp_words_by_ref: command not found
```

**Comment by srsudar** [comment]
And I should add that I am on a Mac, using `bash` 4.2 according to that copied code above.

**Comment by imomaliev** [comment]
Looks like you have older version of bash-completion. `_get_comp_words_by_ref` was added in 1.2 https://github.com/scop/bash-completion/blob/master/CHANGES#L1166.

**Comment by srsudar** [comment]
Yep you're right. Thanks for the help.

I suggest putting something in the README about a dependency on bash_completion, or on a preferred distribution mechanism that might take care of this automatically. To the best of my knowledge (at least on Mac), bash_completion has to be installed and managed by the user. When I encountered the error and didn't see any setup instructions in the README, I assumed all I had to do was source it.
